tasks/internal/update-fbc-catalog-task/update-fbc-catalog-task.yaml

```diff
@@ -127,11 +127,12 @@ spec:
             # than the upstream catalog index.
             # in case the new catalog index is older than the upstream, a new build is
             # required to assure the catalog integrity.
-            if [ "${newCatalogCreatedDate}" -gt "${upstreamCatalogCreatedDate}" ]; then
-              echo "${build}"
-              exit
+            if [ "${newCatalogCreatedDate}" -lt "${upstreamCatalogCreatedDate}" ]; then
+              echo ""
```

Comment:
I don't think you need to print anything at all. This will print a new line, but when you do `build=$(check_previous_build)` the new line gets trimmed (and then you check if `$build` is empty).

Reply:
cool. echo removed.